### PR TITLE
Schedule: add support for priority, units, tags

### LINKED ISSUE #308: Complete schedule API
Scrapy Cloud's `run.json` API endpoint supports specifying the number of SC units to use, and adding tags to the jobs. These options should also be available in `shub schedule`.

### PR DESCRIPTION
**Pull request by elacuesta**
My two cents to fix #308

### CODE REVIEW

File: shub/schedule.py [hunk 1]
```diff
@@ -43,17 +43,24 @@
 @click.command(help=HELP, short_help=SHORT_HELP)
 @click.argument('spider', type=click.STRING)
 @click.option('-a', '--argument',
-              help='spider argument (-a name=value)', multiple=True)
+              help='Spider argument (-a name=value)', multiple=True)
 @click.option('-s', '--set',
-              help='job-specific setting (-s name=value)', multiple=True)
-def cli(spider, argument, set):
+              help='Job-specific setting (-s name=value)', multiple=True)
+@click.option('-p', '--priority', default=2,
```
Comment (vshlapakov) [inline shub/schedule.py]: Let's add int type check for `priority`/`units` options?

File: shub/schedule.py [hunk 2]
```diff
@@ -69,12 +76,15 @@ def cli(spider, argument, set):
 
 
 def schedule_spider(project, endpoint, apikey, spider, arguments=(),
-                    settings=()):
+                    settings=(), priority=2, units=1, tag=()):
```
Comment (vshlapakov) [inline shub/schedule.py]: I'd put the default values into constants, like `DEFAULT_PRIORITY` etc, to define it only once.

File: shub/schedule.py (hunk 1 above)
Comment (elacuesta) [reply shub/schedule.py]: `int` is assumed because of the default value, but I agree that _explicit is better than implicit_

**Comment by elacuesta** [comment]
Hey Viktor, thanks for the review. I fixed a few small things but I still need to update the tests, I'll do it shortly.
Thanks again for your time!

**Comment by elacuesta** [comment]
Squashed, many thanks @vshlapakov 🙌

**Comment by vshlapakov** [comment]
@elacuesta :+1: It's you who did all the work :slightly_smiling_face:

**Pull request merged**